### DISCUSSION TITLE
There's any possibility to use those notebook codes from DreamBooth to train local?

### DISCUSSION THREAD

**Discussion by meunumerotim333** [TheLastBen/fast-stable-diffusion#2544] [General]
This answer is direct for Ben, I'm a tottally noob. But I would like to know if is possible to use your simple codes to train models on DreamBooth on our own PC. Because I'm a bit scary if Colab starts to block Dreambooth on Free Colab too or happens something that you stop to update the repositories on the future.

So would be nice to know if have someway to use those codes on local. Just for security on the future that we never know what will happen.

**Comment by TheLastBen** [comment]
the code will always be available online, I'm sure there will always be a solution in case of an unfortunate event
